spec/hot.spec.js

```diff
@@ -11,6 +11,10 @@ const webpack = require('webpack');
 const rimraf = require('rimraf');
 const WebpackRecompilationSimulator = require('webpack-recompilation-simulator');
 const HtmlWebpackPlugin = require('../index.js');
+const webpackMajorVersion = Number(require('webpack/package.json').version.split('.')[0]);
+if (isNaN(webpackMajorVersion)) {
+  throw new Error('Cannot parse webpack major version');
+}
```

Comment:
You could use `assert` here https://nodejs.org/api/assert.html#assert_assert_value_message

```suggestion
asset (!isNaN(webpackMajorVersion), 'Cannot parse webpack major version');
```